app/src/main/java/org/openedx/app/AppRouter.kt

```diff
@@ -108,10 +108,14 @@ class AppRouter : AuthRouter, DiscoveryRouter, DashboardRouter, CourseRouter, Di
 
     override fun clearBackStack(fm: FragmentManager) {
```

Comment:
overall looks good to me, it will be great if we can update the `navigateToMain` and `navigateToWhatsNew` as well

Reply:
done